posthog/permissions.py

```diff
@@ -166,12 +168,8 @@ class TeamMemberAccessPermission(BasePermission):
 
     message = "You don't have access to the project."
 
-    def has_permission(self, request, view) -> bool:
-        try:
-            team = view.team
-        except Team.DoesNotExist:
-            return True  # This will be handled as a 404 in the viewset
```

Comment:
Is this behavior maintained?

Reply:
Will have to check! Not sure, still a work in progress tbh